bin/xbps-create/main.c

```diff
@@ -109,6 +114,9 @@ usage(bool fail)
 	"                      'vi:/usr/bin/vi:/usr/bin/vim foo:/usr/bin/foo:/usr/bin/blah'\n"
 	" --build-options      A string with the used build options\n"
 	" --compression        Compression format: none, gzip, bzip2, lz4, xz, zstd (default)\n"
+	" --chown              Set the ownership of files and directories.\n"
```

Comment:
this doesn't handle paths with spaces... maybe accepting the argument multiple times would be better, or is this not a concern?

Reply:
something else to consider: should `chown -R` be representable?